tasks/task383_matres_classification.json

```diff
@@ -0,0 +1,11564 @@
+{
+    "Contributors": [
+        "Yeganeh Kordi"
+    ],
+    "Source": [
+        "MATRES (http://cogcomp.org/papers/NingWuRo18.pdf), (https://github.com/CogComp/MATRES)"
+    ],
+    "Categories": [
+        "Classification" 
+    ],
+    "Definition": "We want to determine which verb can be anchored in the time axis, i.e., has a well-defined notion of time. You will be given a question and a context separated with a \n, and you have to answer that the given verb can be anchored in time or not. We say a verb can be anchored in the real timeline if and only if an Event happened in the past or is happening now or is unconditionally planned to happen in the future. If the verb is not anchorable, choose one of the No. It's intention/wish/opinion, No. It's hypothetical/condition, No. It's negation, No. It's abstract/non-specific, No. It's static or No. It's recurrent. And then classify your answers into \"yes_its_anchorable\", \"no_its_intentionwishopinion\", \"no_its_hypotheticalcondition\", \"no_its_negation\", \"no_its_abstractnonspecific\", \"no_its_static\" and \"no_its_recurrent\".",
```

Comment:
- I think you can safely drop "We want to determine which verb can be anchored in the time axis, i.e., has a well-defined notion of time." 
- "a verb can be anchored  if and only if an Event ....": this sentence fails to say how the discussed verb and event are connected to each other. 
- About output labels: the description clearly defines what is anchorable and what is not. But I am not sure if the fine-grained labels (intention/wish/opinion, hypothetical/condition, negation, etc.) are clear.

Reply:
Thanks! I updated the definition.